keps/sig-scheduling/3521-pod-scheduling-readiness/README.md

```diff
@@ -979,6 +1036,8 @@ Think about adding additional work or introducing new steps in between
 [existing SLIs/SLOs]: https://git.k8s.io/community/sig-scalability/slos/slos.md#kubernetes-slisslos
 -->
 
+No.
```

Comment:
> Thanks. I took a look at https://github.com/kubernetes/community/blob/master/sig-scalability/slos/slos.md#kubernetes-slisslos, a SLO about the time from pod creation to pod scheduled is not defined officially yet, so the answer is No here.

That's not entirely true, because the SLO is from time a pod is created to when it's observed as ready. Which actually includes scheduling time.

But as I mentioned previously, it depends how we define "pod is schedulable" here. Currently it's defined as
```
[1]By schedulable pod we mean a pod that can be scheduled in the cluster without causing any preemption.
```
Which is too weak. We should update it, because e.g. low-priority pod will not be able to preempt anything, but may not be schedulable due to lack of space in the cluster.

So we have two options basically:
(a) say that pod has to be immediately schedulable (without actions from any other components), [which would directly exclude pods with schedulingGates]
(b) say that pods with schedulingGates may be schedulable, but then we're taking a dependency on components we may not control, which I don't like.

So I'm in favor of (a). This will indeed mean the answer is "no" here, but then requires updating:
https://github.com/kubernetes/community/blob/master/sig-scalability/slos/pod_startup_latency.md

Would you mind sending me a PR to update that along with what I mentioned in (a)?

Reply:
This was the confusion I had earlier. Thank you for the info! :)